src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -439,7 +439,7 @@ void SolverInterfaceImpl:: finalize()
         suffix << _accessorName << ".final";
         exportMesh ( suffix.str() );
         if ( context.triggerSolverPlot ) {
```

Comment:
We can also think about entirely removing this block:

```
if ( context.triggerSolverPlot ) {
...
}
```

**Don't forget to check whether `context.triggerSolverPlot` and corresponding API can be removed**